apps/docs/src/app/platform/component-docs/platform-forms/platform-date-picker/platform-date-picker-examples/platform-date-picker-complexi18n.component.ts

```diff
@@ -0,0 +1,170 @@
+/**
+ * This approach has been taken from core/datepicker implementation.
+ * Some part of code has been modified to integrate platform capabilities.
+ */
+
+import { Component, Injectable, ViewChild } from '@angular/core';
+import { CalendarI18n, FdDate } from '@fundamental-ngx/core';
+import { DatePickerComponent } from '@fundamental-ngx/platform';
+
+import * as moment from 'moment';
+import 'moment/locale/es';
+import 'moment/locale/en-gb';
+import 'moment/locale/de';
+import 'moment/locale/fr';
+import 'moment/locale/bg';
+import 'moment/locale/pl';
```

Comment:
`Moment` and loading moment as a whole and loading locales that are unused all have an impact on the size of the bundle.  We are not using a lot of the functionality of moment so it may be beneficial for us to see about using `date-fns` or something much lighter (default Moment before Gzip is 600kb +

Reply:
currently fundamentals team is using MomentJs. date-fns is not present in package.
@InnaAtanasova @JKMarkowski , Do we need to do this?